src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -28,16 +30,123 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Draw,
 };
 
-#endif
+Vec3f D_80865900 = { 0.0f, 0.0f, 0.0f };
+
+static void (*D_8086590C[5])(EnPart*, PlayState*) = {
+    func_80865390, func_808654C4, 0x00000000, 0x00000000, 0x00000000,
+};
+
+void EnPart_Init(Actor* thisx, PlayState* play) {
+    EnPart* this = THIS;
+}
+
+void EnPart_Destroy(Actor* thisx, PlayState* play) {
+    EnPart* this = THIS;
+}
```

Comment:
```suggestion
void EnPart_Init(Actor* thisx, PlayState* play) {
}

void EnPart_Destroy(Actor* thisx, PlayState* play) {
}
```
No need to include the temp if it's not used.